pkg/build/webhook/github/github_test.go

```diff
@@ -90,7 +90,7 @@ func TestWrongContentType(t *testing.T) {
 	client := &http.Client{}
 	req, _ := http.NewRequest("POST", server.URL+"/build100/secret101/github", nil)
 	req.Header.Add("Content-Type", "application/text")
-	req.Header.Add("X-Github-Event", "ping")
+	req.Header.Add("X-GitHub-Event", "ping")
```

Comment:
I don't know which way GitHub sends it. They document it as "Github" as @csrwng pointed out. Prior to this PR we had both X-GitHub-Event and X-Github-Event -- let's have only one. Which one?